src/textual/widgets/_button.py

```diff
@@ -100,3 +205,93 @@ async def on_click(self, event: events.Click) -> None:
     async def on_key(self, event: events.Key) -> None:
         if event.key == "enter" and not self.disabled:
             await self.emit(Button.Pressed(self))
+
+    @staticmethod
```

Comment:
shouldn't this be a `@classmethod`, so sub-classes of this `Button` class could re-use these helper constructors? :slightly_smiling_face:

Reply:
Changed